src/update.ts

```diff
@@ -327,17 +324,7 @@ export class UpdateQuery<
 
 export const makeUpdate =
   (queryExecutor: QueryExecutorFn) =>
-  <T extends Table<any, any>>(table: T) => {
-    const getTableStringToken = (table: Table<any, any>) => {
-      if (table.getOriginalName()) {
-        return new StringToken(
-          `${wrapQuotes(table.getOriginalName())} ${wrapQuotes(table.getName())}`,
-        );
-      }
-
-      return new StringToken(wrapQuotes(table.getName()));
-    };
```

Comment:
This was removed because it was not necessary.  `...table.toTokens()` accomplishes the same result.